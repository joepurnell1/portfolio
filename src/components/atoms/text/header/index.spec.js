import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('Square Flex Item component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <Header>What is love?</Header>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render with custom colour as expected', () => {
    const render = renderer.create(
      <Header colour="#F00B42">Baby don&apost hurt me</Header>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
