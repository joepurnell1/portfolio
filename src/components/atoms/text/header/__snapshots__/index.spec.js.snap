// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render default as expected 1`] = `
<h1
  className="sc-bdVaJa fxUZL"
>
  What is love?
</h1>
`;

exports[`Header component should render with custom colour as expected 1`] = `
<h1
  className="sc-bdVaJa heXFlR"
>
  Baby don&apost hurt me
</h1>
`;
